src/Deck1.java

```diff
@@ -0,0 +1,95 @@
+import java.util.ArrayList;
+import java.util.Collections;
+
+/**
+ * {@code Deck} represented as a {@link java.util.ArrayList java.util.ArrayList}
+ * with implementations of primary methods.
+ *
+ * @convention Every element of $this.rep is a Card.
+ * @correspondence this = $this.rep, where first element of this.rep is bottom
+ *                 of deck and last element of this.rep is top of deck.
+ */
+public final class Deck1 extends DeckSecondary {
+
+    /*
+     * Private members --------------------------------------------------------
+     */
+
+    /**
+     * Representation of {@code this}.
+     */
+    private ArrayList<Card> rep;
+
+    @Override
+    public Deck newInstance() {
+        return new Deck1();
```

Comment:
This is probably fine, but there is a preferred way to do it in previous projects.